renovate.json

```diff
@@ -3,6 +3,12 @@
     "config:base"
```

Comment:
IIRC, we needed to pick and choose from things included in config:base to all the versionScheme for guava to take effect: https://github.com/googleapis/java-shared-dependencies/blob/9f844c4b86cfd5e228a1d3b215726d632c21abf6/renovate.json#L2-L12